easybuild/easyconfigs/c/CondaCreate/condacreate-1.0.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_CondaCreate'
+
+name = "condacreate-module"
```

Comment:
That's indeed my main concern, sticking to the EasyBuild way in the central repo. Of course you can leverage the functionality as you please using your own set of easyconfigs. :)

The example easyconfig(s) we include here to use the `Conda` generic easyblock (which we're working towards, I think) should preferably be strict w.r.t. versions, and reflect well what can be done with the `Conda` easyblock...

Also, I would probably include `-Anaconda2-4.0.0` as a `versionsuffix`, to make it clear that the installation is done on top of Anaconda (and which version of Anaconda), so it can be distinguished from a compile-from-source installation with a particular toolchain, as is usually done with EasyBuild.